gear-programs/vft-gateway/src/services/mod.rs

```diff
@@ -31,6 +31,7 @@ pub struct VftGatewayData {
     admin: ActorId,
     receiver_contract_address: H160,
     vara_to_eth_token_id: HashMap<ActorId, H160>,
```

Comment:
I suppose it's obsolete now

Reply:
why? The mapping is usedn in `transfer_vara_to_eth` service.